src/arm/ffi.c

```diff
@@ -566,8 +566,13 @@ ffi_prep_closure_loc (ffi_closure * closure,
   config[1] = closure_func;
 #else
   memcpy (closure->tramp, ffi_arm_trampoline, 8);
+#if defined (__QNX__)
+  msync(closure->tramp, 8, 0x1000000);	/* clear data map */
+  msync(codeloc, 8, 0x1000000);	/* clear insn map */
```

Comment:
@rth7680 had a different suggestion here: https://github.com/frida/libffi/pull/2.  Not sure how strongly he feels about that; my inclination would be to say this approach is ok enough.

However I don't understand the choice of 0x1000000 here.  Are the various `MS_` constants not available?